euphonic/spectra.py

```diff
@@ -759,39 +810,14 @@ def broaden(self: S2D, x_width: Optional[Quantity] = None,
         ValueError
             If shape is not one of the allowed strings
         """
-        if shape == 'gauss':
-            if x_width is None:
-                xsigma = 0.0
-            else:
-                xsigma = self._gfwhm_to_sigma(
-                    x_width, self.get_bin_centres('x'))
-            if y_width is None:
-                ysigma = 0.0
-            else:
-                ysigma = self._gfwhm_to_sigma(
-                    y_width, self.get_bin_centres('y'))
-            z_broadened = gaussian_filter(
-                self.z_data.magnitude, [xsigma, ysigma],
-                mode='constant')
-        elif shape == 'lorentz':
-            z_broadened = self.z_data.magnitude
-            if x_width is not None:
-                x_broadening = _distribution_1d(
-                    self.get_bin_centres('x').magnitude,
-                    x_width.to(self.x_data_unit).magnitude,
-                    shape=shape)
-                z_broadened = correlate1d(
-                    z_broadened, x_broadening, mode='constant', axis=0)
-            if y_width is not None:
-                y_broadening = _distribution_1d(
-                    self.get_bin_centres('y').magnitude,
-                    y_width.to(self.y_data_unit).magnitude,
-                    shape=shape)
-                z_broadened = correlate1d(
-                    z_broadened, y_broadening, mode='constant', axis=1)
-        else:
-            raise ValueError(
-                f"Distribution shape '{shape}' not recognised")
+        bin_centres = [self.get_bin_centres(ax).magnitude for ax in ['x', 'y']]
+        bwidths = [None]*2
```

Comment:
we do have space to write `bin_widths` here and it would be nicer to read

Reply:
Done